sdk/communication/azure-communication-identity/azure/communication/identity/_communication_identity_client.py

```diff
@@ -75,15 +74,6 @@ def from_connection_string(
         :param str conn_str: A connection string to an Azure Communication Service resource.
         :returns: Instance of CommunicationIdentityClient.
         :rtype: ~azure.communication.identity.CommunicationIdentityClient
-
-        .. admonition:: Example:
-
-            .. literalinclude:: ../samples/identity_samples.py
```

Comment:
Did these samples not exist?

Reply:
There wasn't a specific one I found for connection_string and seeing as we decided to show the entire auth sample under the client in the docs, this felt redundant.